### PR TITLE
Update dependencies and switch to ChomeHeadless

### PR DESCRIPTION
**Pull request by chad-configit**
image

💥 

I've noticed coverage doesn't work, but that appears to be a pre-existing problem.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,22 +6,23 @@
...
+    "karma-jasmine": "^1.1.1",
+    "karma-junit-reporter": "1.2.0",
     "karma-ng-html2js-preprocessor": "^1.0.0",
-    "karma-phantomjs-launcher": "^1.0.1",
-    "karma-teamcity-reporter": "^1.0.0",
-    "phantomjs-prebuilt": "^2.1.12",
-    "rimraf": "2.5.4"
+    "karma-phantomjs-launcher": "^1.0.4",
+    "karma-teamcity-reporter": "^1.1.0",
+    "phantomjs-prebuilt": "^2.1.16",
```
Comment (AButler) [inline package.json]: Do you still need Phantom if you are switching to ChromeHeadless?

Comment (chad-configit) [reply package.json]: Good spot. Will remove.

**Comment by chad-configit** [comment]
Should be good to go now

**Comment by chad-configit** [comment]
> chad-configit reopened this just now

I hate you, `Close and comment` button.

**Pull request merged**